sklearnex/tests/test_run_to_run_stability.py

```diff
@@ -119,8 +120,15 @@ def _run_test(estimator, method, datasets):
         str(i): i
         for i in [
             SVC(),
-            KMeans(),
-            KMeans(init="random"),
+            *(
+                []
+                if not daal_check_version((2024, "P", 700))
+                else [
+                    KMeans(),

```

Comment:
Hard to read, it's better to move condition to highest level.

Reply:
updated, please see if this is better